src/routes/(app)/namespaces/[namespace]/archival/_workflow-filters.svelte

```diff
@@ -81,28 +97,32 @@
     >
       <FilterInput
         parameter="workflow-id"
-        name="Workflow ID"
+        name={translate('workflow-id')}
         value={workflowIdFilter}
       />
       <FilterInput
         parameter="workflow-type"
-        name="Workflow Type"
+        name={translate('workflow-type')}
         value={workflowTypeFilter}
       />
-      <FilterSelect label="Time Range" parameter="time-range" value="24 hours">
+      <FilterSelect
+        label={translate('workflows', 'time-range')}
+        parameter="time-range"
+        value="24 hours"
+      >
         {#each durations as value}
```

Comment:
Have we aligned on an approach to durations yet (e.g. hrs/mins/days)?

Reply:
Not yet, I think we'll need to refactor some components a little, as I don't think we can call `translate()` outside of a svelte component.

Reply:
like the approach to translate them?

Reply:
Yeah, we might have to make these i18n keys, https://github.com/temporalio/ui/blob/main/src/lib/utilities/to-duration.ts#L7-L37 but then there are other utils that use those files, so they'd need to be refactored I think. There might be a simpler way to do it though